shell/browser/api/electron_api_session.cc

```diff
@@ -120,27 +121,25 @@ struct ClearStorageDataOptions {
 };
 
 uint32_t GetStorageMask(const std::vector<std::string>& storage_types) {
+  static constexpr electron::StringLookup<uint32_t, 9> typemap{{{
```

Comment:
Having to specify the number of elements is a bit of a bummer, can we infer it?

It's also a bit surprising that we need three sets of curly braces here.

Reply:
We [can](https://stackoverflow.com/questions/6114067/how-to-emulate-c-array-initialization-int-arr-e1-e2-e3-behaviou) infer it. It's cleaner in c++17, but it's doable in 14. If that's the make-or-break threshold for this PR then I'll make that change.

The curly braces are (I believe) necessary due to C++'s brace elision rule. std::pair isn't an aggregate, so it needs extra braces to be list-initialized. This holds true for the pairs _and_ the array. I agree it's an  unusual look :)